app/models/placements/mentor.rb

```diff
@@ -0,0 +1,16 @@
+# == Schema Information
+#
+# Table name: mentors
+#
+#  id         :uuid             not null, primary key
+#  first_name :string           not null
+#  last_name  :string           not null
+#  trn        :string           not null
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+class Placements::Mentor < Mentor
+  default_scope { joins(:mentor_memberships).distinct }
+  has_many :mentor_memberships
+  has_many :schools, through: :mentor_memberships
+end
```

Comment:
This might be OCD, but:

```ruby
class Claims::Mentor < Mentor
  has_many :mentor_memberships
  has_many :schools, through: :mentor_memberships

  default_scope { joins(:mentor_memberships).distinct }
end
```

😆

Reply:
@Nitemaeric possibly a bit 😂 

But it's worth pointing out that there's a RuboCop rule specifically for this: [Layout/ClassStructure](https://docs.rubocop.org/rubocop/cops_layout.html#layoutclassstructure)

If you think the structure of this class is worth changing, would it be worth switching on this rule so there's a consistent style across the board? (presumably out of scope for this PR though)

Reply:
The RuboCop is definitely for another PR :)

Happy to keep them different, but it just jumped out when going through the code.

Reply:
0dfcc333083a61294db041c68cc2b7f4627cf2cb